.idea/.name

```diff
@@ -0,0 +1 @@
+ms-SG.js
```

Comment:
I see you're an Jetbrains user?

Reply:
oh haha. I was just testing Jetbrains. Actually, I am VSCode user :)